### PR TITLE
SNO-181-Update-install-readme

### PR DESCRIPTION
**Pull request by caseylitton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,13 +47,17 @@ SnoVault JSON-LD Database Framework
...
     ```
-    $ brew tap AdoptOpenJDK/openjdk
-    $ brew cask install adoptopenjdk8
+    $ brew install adoptopenjdk8
     ```
 
 1. Elasticsearch 5.x
     ```
-    $ brew install elasticsearch@5.6
+    $ https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.6.0.tar.gz
```
Comment (khzlin) [inline README.md]: change $ to sharp sign.

File: README.md [hunk 2]
```diff
@@ -43,13 +47,17 @@ SnoVault JSON-LD Database Framework
...
+    $ tar -xvf ~/Downloads/elasticsearch-5.6.0.tar.gz -C /usr/local/opt/
```
Comment (khzlin) [inline README.md:66]: no ".gz"

Comment (caseylitton) [reply README.md:66]: what do you mean?  The extension for the url and file is .tar.gz on the url and download?

Comment (khzlin) [reply README.md:66]: oh I think my browser automatically decompressed it.

**Review by khzlin — APPROVED** [review]

**Pull request merged**